### PR TITLE
PR for Playlist management

### PR DESCRIPTION
**Pull request by gopalbbharadva**
#### Features:
- Make playlist from home page and single video page.
- remove playlist from playlist page.
- add/remove video to/from playlist.
- closes modal and menu on outside click.

Live link : https://deploy-preview-10--vframe.netlify.app/

P.S: This PR is Big, as I fix all small small features , which are not fixed before. ( check features on UI, if it's working or not.)

### CODE REVIEW

File: src/frontend/components/playlistmodal/playlistitem/PlaylistItem.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                {title}
+              </label>
+            </div>
+          );
+        }
+      )}
+    </>
+  );
+};
+// videos.some((item) => item._id === videoId)
```
Comment (Chakravarthi0) [inline src/frontend/components/playlistmodal/playlistitem/PlaylistItem.jsx:61]: You can remove this, if not needed

File: src/frontend/components/videocard/videocard.css [hunk 2]
```diff
@@ -52,13 +53,14 @@
...
-  right: -2rem;
+  list-style: none;
+  top: 8px;
+  right: -1rem;
   width: max-content;
   background-color: var(--Primary_Color_100);
   position: absolute;
   padding: 0.8rem;
-  z-index: 999;
+  /* z-index: 1; */
```
Comment (Chakravarthi0) [inline src/frontend/components/videocard/videocard.css:63]: this comment as well

File: src/frontend/helperfunctions/getVideo.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const getVideo = (videoId) => `https://www.youtube.com/embed/${videoId}`;
```
Comment (Chakravarthi0) [inline src/frontend/helperfunctions/getVideo.js:1]: You have made a function to get the video, which is nice.

**Review by Chakravarthi0 — APPROVED** [review]
code is clean
app is working as expected
good to merge

**Review by MiheerTamkhane — APPROVED** [review]
Code is looking good and Well structured. 
Good to merge for me....

**Pull request merged**